src/main/java/io/github/perrymant/moneymaker/Application.java

```diff
@@ -1,20 +1,31 @@
 package io.github.perrymant.moneymaker;
 
 class Application {
-    private final Budget budget = new Budget(new Balance(), new TransactionMaker().getTransactions());
+    private final DefaultBudget budget = new DefaultBudget(new Balance(), new TransactionMaker().getTransactions());
     private final FileReader reader = new FileReader();
-
     private final Logger logger;
 
+    private static final String HELP_TXT = "moneymaker-help.txt";
+    private static final String ERROR = "" +
+            "Error: Invalid argument:\n" +
+            "$ moneymaker help\n" +
+            "for more info.";
+
     Application(Logger logger) {
         this.logger = logger;
     }
 
     void start(String[] args) {
```

Comment:
Ideally we'd only have one logger call, so can we refactor this method into two please:

- One method that returns a string that should be given to the logger
- One method that gets the string from the above method, and passes it to the logger.

Reply:
I'm not sure if I was able to refactor this how you stated it...